### DISCUSSION TITLE
LaTeX Error: File bytefield.sty not found

### DISCUSSION THREAD

**Discussion by willwray** [microsoft/ifc-spec#6] [General]
FYIs
Following the readme instructions, on my setup (Fedora, texlive)
`> pdflatex ifc`
terminated with this error and stopped on it's own command line
(unfamiliar, I quit with a couple of CTRL-C's I think).

Solved by installing with `sudo dnf install texlive-bytefield`

(Might be worth enabling the Wiki, or the Discussions feature if available.)

**Comment by gdr-at-ms** [comment]
> (Might be worth enabling the Wiki, or the Discussions feature if available.)

The Discussion feature is now enabled.
